.pre-commit-config.yaml

```diff
@@ -1,6 +1,8 @@
 default_language_version:
   python: python3.9
 
+exclude: tests/acceptance/external_studies/
```

Comment:
Ah, I wish I'd known this was possible earlier! I ended up taking the nuclear option to work around this elsewhere:
https://github.com/opensafely-core/databuilder/blob/5adc1735960748d3334979f5437a5c1da1b2f9dc/.github/workflows/update-external-studies.yml#L26-L27